### PR TITLE
SHOR-16: Fixed membership issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
This PR does the following changes for membership,
https://s.nimbusweb.me/share/1658636/rp6ka8qg1dq3cys790q4
https://s.nimbusweb.me/share/1658592/f7egeigu2x8h2hhc1r9p

# Screenshots
**Before**
find memberships training 1

**After**
find memberships training

**Before**
find memberships training 2

**After**
civimember training 13

# Note
This PR needs to have code PR merged to make proper structure [link of core](https://github.com/civicrm/civicrm-core/pull/12822)

# Comments
- BackstopJS tested
- Stylelint tested

### CODE REVIEW

File: scss/civicrm/contact/pages/_memberships.scss [hunk 1]
```diff
@@ -423,6 +423,12 @@
       width: 105%;
     }
 
+    &.no-border {
+      margin-left: -20px;
+      margin-right: -20px;
+      width: 105%;
```
Comment (reneolivo) [inline scss/civicrm/contact/pages/_memberships.scss]: 105%? would this break if the screen shrinks or expands? Shouldn't it be something like `width: calc(100% - 40px);` ?

File: scss/civicrm/contact/pages/_memberships.scss [hunk 2]
```diff
@@ -522,17 +532,37 @@
         }
       }
 
-      tr td.label {
-        background: none !important;
-        padding: 10px 30px !important;
+      tr {
+        th {
+          padding: 10px 20px !important;
```
Comment (reneolivo) [inline scss/civicrm/contact/pages/_memberships.scss]: can't we avoid adding `!important` ? is there no way to prioritize these rules without using `!important` ?

File: scss/civicrm/contact/pages/_memberships.scss (hunk 1 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_memberships.scss]: Nope: Other classes in pages are fixed like this. so, it's not breaking. see few lines above for the same.

File: scss/civicrm/contact/pages/_memberships.scss (hunk 2 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_memberships.scss]: Nope! it is not working without it, due to many already previous !important s.

Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_memberships.scss]: Fixed

**Pull request merged**